src/components/molecules/Tag/Tag.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react";
+import classNames from "classnames";
+import { Close, TagOutline, WarningFill, ErrorAlertFill } from "@geneui/icons";
+
+// Styles
+import "./Tag.scss";
+
+// Components
+import Button from "../../atoms/Button/Button";
```

Comment:
Please reorder imports, make the last import // Styles

Reply:
Done